examples/mini-compile/LICENSE

```diff
@@ -0,0 +1,217 @@
+Copyright Digital Asset 2018-2019.
```

Comment:
I think this (and the other `LICENSE`s) should be `-2022`, no?

```suggestion
Copyright Digital Asset 2018-2022.
```

Reply:
good catch